controllers/whoToFollow/whoToFollow.js

```diff
@@ -83,7 +93,6 @@ module.exports = async (req, res) => {
       for (const indexUserTopic in userTopicFollower) {
         const userTopic = userTopicFollower[indexUserTopic];
         delete userTopic.human_id;
-        if (tempUsers?.length >= 10) break;
         if (
           userTopic.topic_id === topic.topic_id &&
           userTopic.user_id !== process.env.BETTER_ADMIN_ID
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you're deleting the `human_id` property from the `userTopic` object. If `userTopicFollower` is a reference to an object stored elsewhere in your application (e.g., cached data), this could unintentionally affect other parts of your application that rely on the `human_id` property. Consider creating a copy of the object before modifying it.
```diff
-         const userTopic = userTopicFollower[indexUserTopic];
-         delete userTopic.human_id;
+         const userTopic = {...userTopicFollower[indexUserTopic]};
+         delete userTopic.human_id;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->